experiments/generic/drain_node/drain_node_ansible_logic.yml

```diff
@@ -0,0 +1,128 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars:
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    c_experiment: "node-drain"
+    data_persistance: "{{ lookup('env','DATA_PERSISTENCY') }}"
+    liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+
+  tasks:
+    - block:
+
+        ## PRE-CHAOS APPLICATION LIVENESS CHECK
+        - include_tasks: /utils/common/application_liveness_check.yml
+          when: liveness_label != ''
+
+        ## RECORD START-OF-TEST IN CHAOS RESULT CR
+        
+        - include_tasks: /utils/runtime/create_testname.yml
+
+        - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+          vars:
+            status: 'SOT'
+            namespace: "{{ a_ns }}"
+
+        ## DISPLAY APP INFORMATION 
+ 
+        - name: Display the app information passed via the test job
+          debug: 
+            msg: 
+              - "The application info is as follows:"
+              - "Namespace    : {{ a_ns }}"
+              - "Label        : {{ a_label }}"
+
+        - name: Verify that the AUT (Application Under Test) is running
+          include_tasks: "/utils/common/status_app_pod.yml"
+          vars:       
+            delay: 5
+            retries: 60
+
+        - name: Get application pod name
+          shell: >
+            kubectl get pods -n {{ a_ns }} -l {{ a_label }} --no-headers
+            -o=custom-columns=NAME:".metadata.name"
+          args:
+            executable: /bin/bash
+          register: app_pod_name
+
+        - name: Generate unique string for use in dbname
+          shell: echo $(mktemp) | cut -d '.' -f 2
+          args:
+            executable: /bin/bash
+          register: uniqstr
+
+        - name: Create some test data in the mysql database
+          include_tasks: "/utils/apps/mysql/mysql_data_persistence.yml"
```

Comment:
@shubhamchaudhary  can you use jinja template to use different data_persistence check utils for different application.